assets/js/app.js

```diff
@@ -0,0 +1,59 @@
+(function() {
+var weapon1 = $('.weapon1');
+var weapon2 = $('.weapon2');
+var special = $('.special');
+
+var heroes = [Mal];
+var enemies = [allianceSoldier];
+function Game() {
+  this.on(takeTurnWithWeapon, function() {
+    enemies[1].trigger(attacked);
+  });
+
+}
+
+//need to remember to remove characters from array once defeated
+// end turn after attack function
+
+var takeTurnWithWeapon = function(weaponIndex) {
+
+};
+
+weapon1.on('click', function() {
+  takeTurnWithWeapon(0);
```

Comment:
Try changing this to `game.attack(0)`. Then in `Game.prototype.attack` take in a weapon index. then update the click below to fit with the same function `game.attack(1)`.